README.md

```diff
@@ -3,8 +3,44 @@
 If you are the kind of person who likes thinking _long-term_ and _high-impact_
 this repo is for you!
 
-**D**o **W**hat **Y**ou **L**ove is _not_ a 3-month stepping-stone project ...
+**D**o **W**hat **Y**ou **L**ove is _not_ a stepping-stone project, we're in it for the long-haul. <br />
+We are making sure that the direction we're heading in is clear to everyone who wants to know.
 
-We _very much_ want/need help on this so if you have _time_
+### Our Goal/Mission
+
+> Empower people to sustainably unlock their potential and pursue their passion through technology.
+
+So what does this encompass? Any number of things that will need to be split out into measurable goals:
++ Living a happy life
```

Comment:
I mean... _everything_ is controversial in some way.
Living a happy life shouldn't imply a pursuit of happiness per se, but a goal to aspire to - what makes each person happy is different. The key is to improve the little things that add up to your own personal sense of what makes you happy.